@here/harp-omv-datasource/lib/OmvData.ts

```diff
@@ -221,9 +219,15 @@ export class GeometryCommands {
                         yCoords.push(currY);
                     }
 
-                    const longitude = west + currX * longitudeScale;
-                    const latitude = tile2lat(top + currY, tileKey.level + N);
-                    commands.push({ kind, latitude, longitude });
+                    const R = EarthConstants.EQUATORIAL_CIRCUMFERENCE;
```

Comment:
Same as above!?

Reply:
I guess if we really want we could export some more function from TileKey/SubdivisionScheme/TilingScheme to deal with this but I'm not sure it is worth doing it. Also, it will create a bit of an additional APIs. I don't have strong feelings about it, so if we want I can export helpers to deal with the normalized mercator space and higher subdivisions.